kpi/models/asset_version.py

```diff
@@ -74,3 +74,20 @@ def __str__(self):
             self.asset.uid, self.uid,
             self.date_modified.strftime('%Y-%m-%d %H:%M'),
             ' (deployed)' if self.deployed else '')
+
+    def save(
+        self,
+        force_insert=False,
+        force_update=False,
+        using=None,
+        update_fields=None,
+    ):
+        if self.pk:
```

Comment:
Why wouldn't it be set on creation? And if so, it may be cleaner to use `auto_now=True` in the model - that's a pretty typical way to build modified dates.

Reply:
Because on the creation it will be pick the default, so no need to override it again. But I guess it does not hurt anyway.
AFAIK, we try to avoid `auto_now` (and `auto_now_add`) as much as possible because it alters the field even if we don't want to, e.g. running a management command that updates the model without changing dates. 
We can discuss about this with @jnm if needed.

Reply:
If we avoid auto_now - I'd probably just drop the if statement - I think it's more readable to see that it should always be set. I can't imagine a performance difference here. 

auto_now

> The field is only automatically updated when calling [Model.save()](https://docs.djangoproject.com/en/4.2/ref/models/instances/#django.db.models.Model.save). The field isn’t updated when making updates to other fields in other ways such as [QuerySet.update()](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#django.db.models.query.QuerySet.update), though you can specify a custom value for the field in an update like that.

auto_now is one of those Python only implementations that will would cause a "useless" migration. I certainly don't want to make a slow migration that does absolutely nothing. Do you know if it is indeed slow, maybe postgres would be smart enough to not rewrite tons of disk. Could discuss more if desired. If we could avoid slow migrations, then I think auto_now is better. But not worth a painful migration. Do we have any existing policy for small things like help_text? My personal and maybe controversial policy is to not make "useless" migrations. Let django raise a warning about it.